### DISCUSSION TITLE
[M-Z] (12 Febbraio 2019) Esercizio 4

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#440] [Esami & Esoneri M-Z]
## Traccia

<img width="871" alt="image" src="https://github.com/user-attachments/assets/34c09ff1-c242-462c-8ad0-5bc62a1bba4a">

**Comment by rimaout** [comment]
<img width="500" alt="image" src="https://github.com/user-attachments/assets/25f94201-1d4f-4585-a2e4-e488107d4467">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190212.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190212.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/811fde52-b7ed-48dd-8619-3b1a9bbd3a60)
